fastapi_cache/backends/inmemory.py

```diff
@@ -20,13 +20,14 @@ class InMemoryBackend(Backend):
     def _now(self) -> int:
         return int(time.time())
 
-    def _get(self, key: str):
+    def _get(self, key: str) -> Value | None:
```

Comment:
I think `Optional[Value]` should be used to ensure compatibility.

Reply:
Sure, thank you!